tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -830,58 +831,87 @@ public function test_non_sk_user_scripts_not_enqueued_for_sk_users() {
 			)
 		);
 
-		do_action( 'enqueue_block_editor_assets' );
```

Comment:
This action trigger wasn't actually doing anything, I've fixed these tests to properly verify the assets are enqueued or not.